serverlessworkflow/modules/ROOT/pages/tooling/serverless-logic-web-tools/serverless-logic-web-tools-openshift-integration.adoc

```diff
@@ -7,37 +7,37 @@
 You can integrate your {product_name} project with Red Hat OpenShift. OpenShift is an enterprise-ready Kubernetes container platform, enabling your {product_name} projects to be deployed and tested online.
 
 [[proc-setting-kie-sandbox-extended-services-serverless-logic-web-tools]]
-== Setting KIE Sandbox Extended Services
+== Setting Extended Services
```

Comment:
```suggestion
== Setting extended services
```

Reply:
"Extended Services" is the name of a tool that we have so I will keep it with uppercase.